bundles/org.openhab.binding.boschindego/src/main/resources/OH-INF/i18n/boschindego.properties

```diff
@@ -56,3 +56,9 @@ channel-type.boschindego.statecode.state.option.774 = Returning to Dock
 channel-type.boschindego.statecode.state.option.775 = Returning to Dock - Lawn complete
 channel-type.boschindego.statecode.state.option.775 = Returning to Dock - Relocalising
 channel-type.boschindego.textualstate.label = Textual State
+
```

Comment:
You should also have new entries for added state options.

Reply:
Updated, thanks.